ui/src/utils/debounce.ts

```diff
@@ -1,12 +1,20 @@
 function debounce<C extends (...args: Parameters<C>) => ReturnType<C>>(
   callback: C,
-  wait: number
+  wait: 0,
```

Comment:
My bad. this is small typo here, it should default to `0` not have a type of `0`:
```suggestion
  wait = 0,
```